rotkehlchen/api/v1/schemas.py

```diff
@@ -188,19 +190,29 @@ def make_ethereum_transaction_query(  # pylint: disable=no-self-use
     ) -> Dict[str, Any]:
         address = data.get('address')
         order_by_attribute = data['order_by_attribute'] if data['order_by_attribute'] is not None else 'timestamp'  # noqa: E501
+        protocols, asset = data['protocols'], data['asset']
+        if not protocols:  # If protocols == []
+            protocols = None
```

Comment:
I believe lefteris will agree that here is better to be explicit and why to write it in the comment when you can write it as code. I would use 
```suggestion
        if len(protocols) == 0:
            protocols = None
```

This would prevent (maybe not here but in other scenarios) possible errors that come from invalid types for protocols

Reply:
I used `not protocols` here because `protocols` can be either `None` or `[]` so `len(protocols)` might fail. But I surely can change it to a bigger explicit condition like `if protocols is None or len(protocols) == 0`